examples/socketio/src/controllers/socketio.controller.ts

```diff
@@ -0,0 +1,103 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-socketio
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {socketio, Socket} from '@loopback/socketio';
+import debugFactory from 'debug';
+
+const debug = debugFactory('loopback:socketio:controller');
+
+/**
+ * A demo controller for socket.io
+ */
+@socketio('/')
```

Comment:
Let's add a comment here to briefly mention the purpose of `@socketio`.

Reply:
`/**
 * A demo controller for socket.io
 * ```ts
 * @socketio('/')
 * ```
 * This specifies the namespace / for this controller
 * Regex or strings are acceptable values for namespace
 */`
Adding info to show this specifies namespace for the controller